src/probnum/random_variables/_normal.py

```diff
@@ -139,6 +139,7 @@ def __init__(
         self._cov = cov
 
         self._compute_cov_cholesky: Callable[[], _ValueType] = None
+        self._cov_cholesky_values = cov_cholesky  # recall: None if not provided
```

Comment:
default is None. If provided, the values of the Cholesky factors are stored here. If not, this variable is changed by `self.precompute_cov_cholesky()`

Reply:
Why is this in the plural? Can we have multiple Cholesky decompositions in one Normal random variable?

Reply:
because it is a matrix... somehow `value` felt wrong. I do not have a strong opinion though, so if you prefer "value", I will change it.

Reply:
Personally, I would just call it `_cov_cholesky`, is there anything preventing this naming?

Reply:
Not strictly, there is just so many methods associated with the Cholesky that I thought it did not hurt to clarify that this one holds the array (if it exists). 

Do you still prefer just `_cov_cholesky`? If so, I will change it.

Reply:
I would prefer `_cov_cholesky`, but I usually have a strong opinion on naming, so feel free to ignore it 😃 Nevertheless, here's my reasoning: I feel like the internal variable storing the cached value of the public property `cov_cholesky` should be named `_cov_cholesky` to associate the two with one another.

Reply:
Done